meta-flutter/README.md

```diff
@@ -52,6 +52,32 @@ $ ./tmp/deploy/sdk/poky-glibc-x86_64-core-image-weston-aarch64-qemuarm64-toolcha
 ```
 
 ## Cross-building with bitbake
+### Flutter Engine library
+Flutter Engine recipe is limited to **linux** and **AArch64(ARM64)** targets. Flutter Engine version (commit id / SHA) defined in the recipe is as follows.
+```
+# FYI: Latest as of 2021-05-09.
+ENGINE_VERSION ?= "f5b97d0b23a3905e9b5b69aa873afcb52f550aaf"
+```
+You need to use the following Flutter SDK versions when creating a Flutter project.  
+`Flutter 2.3.0-1.0.pre.269`
+
+Add the following in your `conf/local.conf`, if you want to change the Flutter Engine version. You need to modify <engine_version> appropriately for your Flutter SDK versions:
+```
+ENGINE_VERSION_pn-flutter-engine = <engine_version>
+```
+
+Flutter Engine recipe has three compile options (debug / profile / release modes). The default compile option is `release-mode`. Add the following in your `conf/local.conf`, if you want to change the compile options:
```

Comment:
Flutter Engine is built with release mode by default. If you want to change the build mode, you can change it to add the following in your `conf/local.conf`:

Reply:
I fixed it.